data/data/bootstrap/files/usr/local/bin/bootstrap-pivot.sh.template

```diff
@@ -1,6 +1,16 @@
 {{if .IsOKD -}}
 #!/usr/bin/env bash
 set -euo pipefail
+
+# Exit early if pivot is attempted on SCOS Live ISO
+{{if .IsSCOS -}}
+source /etc/os-release
+if [[ ! $(touch /usr/.test) ]] && [[ ${ID} =~ ^(centos)$ ]]; then
```

Comment:
It's not going to work on any live ISO right?  As we move towards potentially doing https://github.com/openshift/os/issues/799 we'll also want to have this change for OCP/RHEL.

That said, I think the Live ISO case is going to have interesting issues here.  What we probably want to support is a case where we do dynamic overlays of things like kubelet and other tools, without a reboot.

Reply:
Right, that script would fail on any LiveISO, but we run boostrap-pivot on OKD only. The CentOS guard here is because OKD SCOS can start via:
* FCOS in IPI
* LiveISO SCOS in Assisted Installer

Reply:
Right, but I'm saying we're at least potentially going to make the same change for OCP to ship disk images that are pure RHEL content.  (enhancement etc. to be written; no action required now, just noting)

Reply:
Ah, yes, in that case we'll need something similar - and it would be a great opportunity to rework bootstrap-pivot entirely